server/service/src/asset/delete.rs

```diff
@@ -55,7 +56,7 @@ pub fn validate(
     };
 
     if let Some(store_id) = &asset_row.store_id {
-        if ctx_store_id != store_id {
+        if ctx_store_id != store_id && !CentralServerConfig::is_central_server() {
```

Comment:
This is to ensure remote sites can't delete equipment from other stores.

I'm now wondering if it would be important to cover this use case with tests?

Reply:
I would argue the change it to allow central server to delete the remote sites data...
I think it's ok to not have a test case for this for now, although I agree it would be a good idea. It's not easy to put that into a test case though, although I guess we can probably fake being a central server somehow?

The whole area needs a review I think as it's confusing for users to have different behaviour on central server vs remote.